JSDemos/Demos/DataGrid/BatchUpdateRequest/Angular/app/app.component.ts

```diff
@@ -0,0 +1,65 @@
+import { NgModule, Component, enableProdMode } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+import { HttpClient, HttpClientModule } from '@angular/common/http';
+import DxDataGrid from 'devextreme/ui/data_grid';
+import { DxDataGridModule } from 'devextreme-angular';
+import * as AspNetData from 'devextreme-aspnet-data-nojquery';
+
+if (!/localhost/.test(document.location.host)) {
+    enableProdMode();
+}
+
+const URL = 'https://js.devexpress.com/Demos/Mvc/api/DataGridBatchUpdateWebApi';
+
+@Component({
+    selector: 'demo-app',
+    templateUrl: 'app/app.component.html',
+    styleUrls: ['app/app.component.css'],
+    preserveWhitespaces: true
+})
+export class AppComponent {
+    ordersStore: any = [];
+
```

Comment:
```suggestion
    ordersStore: any;
```
Since we are going to create an instance of the CustomStore, it is not necessary to initialize ordersStore as an empty array.